dotCMS/src/main/java/com/dotcms/publisher/util/DependencyManager.java

```diff
@@ -1085,17 +1115,24 @@ private void structureDependencyHelper(final String stInode, final PublisherFilt
 		final Structure structure = CacheLocator.getContentTypeCache().getStructureByInode(stInode);
 
 		// Host Dependency
-		if(!publisherFilter.doesExcludeDependencyClassesContainsType(PusheableAsset.SITE.getType())) {
-			final Host host = APILocator.getHostAPI().find(structure.getHost(), user, false);
-			hosts.addOrClean(structure.getHost(), host.getModDate()); // add the host dependency
+		if (!publisherFilter.doesExcludeDependencyClassesContainsType(PusheableAsset.SITE.getType())) {
+			APILocator
+					.getHostAPI().find(structure.getHost(), user, false)
+					.ifSystemHostLog(
+							this,
+							"Detected SystemHost for host dependency, ignoring it",
```

Comment:
Codacy found an issue: [The String literal "Detected SystemHost for host dependency, ignoring it" appears 4 times in this file; the first occurrence is on line 1,123](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6340082)